### ISSUE TITLE
but offline

### ISSUE THREAD

**Issue by stickylabdev** [dommilosz/Among-us-but-3D#1]
can this be online multiplayer ?

**Comment by dommilosz** [comment]
yes there is branch multiplayer-test we are working on it. Currently there is movement rotation and player and game settings synchronization implemented.

**Comment by dommilosz** [comment]
You can always help with code quality/bug fixes via Pull Requests to speed up the work!

**Comment by stickylabdev** [comment]
cool
